index.js

```diff
@@ -32,7 +35,8 @@ module.exports = {
         'promise/no-promise-in-callback': 'warn',
         'promise/no-callback-in-promise': 'warn',
         'promise/avoid-new': 'warn',
-        'promise/no-return-in-finally': 'warn'
+        'promise/no-return-in-finally': 'warn',
+        'promise/valid-params': 'warn'
```

Comment:
Adding this to the recommended config as a warning. Thoughts on making this an error instead?